_config/ispn.yml

```
@@ -48,24 +48,24 @@ stable:
     jmxdocs: https://docs.jboss.org/infinispan/11.0/apidocs/jmxComponents.html
 
 # Please remove or comment out the following 'unstable' version section if this is not applicable at the moment.
-#unstable:
-#  minor_version: 12.0
-#  version: 12.0.0.Dev01
-#  maven_latest: 12.0.0.Dev01
-#  release_date: 2020/07/03
-#  fixed_issues: https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12310799&version=12344573
-#  codename: N/A
-#  whats_new: Async XSite with conflict resolution, non-blocking everywhere, server security usability, etc...
-#  download:
-#    server: https://downloads.jboss.org/infinispan/12.0.0.Dev01/infinispan-server-12.0.0.Dev01.zip
-#    wfmodules: https://downloads.jboss.org/infinispan/12.0.0.Dev01/infinispan-wildfly-modules-12.0.0.Dev01.zip
-#    docker: quay.io/infinispan/server:12.0.0.Dev01-1
-#  docs:
-#    home: /docs/dev/index.html
-#    contributing: /docs/dev/titles/contributing/contributing.html
-#    javadocs: https://docs.jboss.org/infinispan/12.0/apidocs/
-#    configdocs: https://docs.jboss.org/infinispan/12.0/configdocs/
-#    jmxdocs: https://docs.jboss.org/infinispan/12.0/apidocs/jmxComponents.html
+unstable:
+  minor_version: 12.0
+  version: 12.0.0.Dev01
+  maven_latest: 12.0.0.Dev01
+  release_date: 2020/07/27
+  fixed_issues: https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12310799&version=12344573
```

The link is pointing to 11.0.0.CR1 release notes.
```suggestion
  fixed_issues: https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12310799&version=12346555
```